kong/plugins/statsd/handler.lua

```diff
@@ -15,19 +15,19 @@ local NGX_ERR = ngx.ERR
 local gauges = {
   request_size = function (api_name, message, logger)
     local stat = api_name..".request.size"
-    logger:gauge(stat, message.request.size, 1)
+    logger:timer(stat, message.request.size, 1)
```

Comment:
Statsd takes timer data and creates average and TP stats out of them as opposed to a gauge, which just overrides the previous gauge value. We were hoping to get those stats for our request / response sizes. 

Point 14: https://blog.raintank.io/25-graphite-grafana-and-statsd-gotchas/